app/layout.tsx

```diff
@@ -0,0 +1,18 @@
+import "@/src/styles/global.scss";
+
+export const metadata = {
+  title: "Linkbrary",
+  description: "bokeeey weeklyMission",
+};
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
```

Comment:
이렇게 표현해주는게 더 좋을 것 같습니다~
```suggestion
  children: ReactNode;
```